system-test/vision.ts

```diff
@@ -0,0 +1,158 @@
+/*!
```

Comment:
Let's switch the comment style to be consistent with the rest of the module

Reply:
of course, thank you!